src/System.Security.Permissions/src/System.Security.Permissions.csproj

```diff
@@ -171,16 +181,24 @@
     <Compile Include="System\ServiceProcess\ServiceControllerPermissionEntryCollection.cs" />
     <Compile Include="System\Transactions\DistributedTransactionPermission.cs" />
     <Compile Include="System\Transactions\DistributedTransactionPermissionAttribute.cs" />
+    <Compile Include="System\Web\AspNetHostingPermission.cs" />
+    <Compile Include="System\Web\AspNetHostingPermissionAttribute.cs" />
+    <Compile Include="System\Web\AspNetHostingPermissionLevel.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNetFx)' == 'true'">
     <Reference Include="mscorlib" />
     <Reference Include="System" />
+    <Reference Include="System.Configuration" />
+    <Reference Include="System.Core" />
     <Reference Include="System.Data" />
     <Reference Include="System.Data.OracleClient" />
+    <Reference Include="System.DirectoryServices" />
```

Comment:
@danmosemsft @safern I added System.DirectoryServices, System.Core etc. here but [this link](https://ci3.dot.net/job/dotnet_corefx/job/master/job/windows-TGroup_netfx+CGroup_Release+AGroup_x86+TestOuter_false_prtest/18260/console) shows warnings/errors below:

```
16:17:12 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermission' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:12 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionAccess' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:12 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionAttribute' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:12 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionEntry' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:12 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionEntryCollection' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:12 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Errors were encountered while generating the facade. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:12 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): error : Errors were encountered when generating facade(s). [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14   System.Security.Cryptography.Xml -> D:\j\workspace\windows-TGrou---2a8f9c29\artifacts\bin\System.Security.Cryptography.Xml\netfx-Release\System.Security.Cryptography.Xml.dll
16:17:14 
16:17:14 Build FAILED.
16:17:14 
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.dotnet\sdk\2.1.401\Microsoft.Common.CurrentVersion.targets(2110,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "System". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [D:\j\workspace\windows-TGrou---2a8f9c29\external\test-runtime\XUnit.Runtime.depproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.dotnet\sdk\2.1.401\Microsoft.Common.CurrentVersion.targets(2110,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "System.Core". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [D:\j\workspace\windows-TGrou---2a8f9c29\external\test-runtime\XUnit.Runtime.depproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermission' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionAccess' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionAttribute' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionEntry' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Did not find type 'T:System.DirectoryServices.DirectoryServicesPermissionEntryCollection' in any of the seed assemblies. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): warning : Errors were encountered while generating the facade. [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14 D:\j\workspace\windows-TGrou---2a8f9c29\.packages\microsoft.dotnet.genfacades\1.0.0-beta.18556.6\build\Microsoft.DotNet.GenFacades.targets(98,5): error : Errors were encountered when generating facade(s). [D:\j\workspace\windows-TGrou---2a8f9c29\src\System.Security.Permissions\src\System.Security.Permissions.csproj]
16:17:14     8 Warning(s)
16:17:14     1 Error(s)
```

Reply:
I investigated the problem and found the issue. System.DirectoryServices ref project has a netstandard build configuration, making it compatible with the netfx vertical. So whenever we're building the netfx vertical it is building a netfx reference assembly and binplacing it into the bin\ref\netfx folder which is where we grab the netfx references from. Since this types are not exposed in our netstandard reference assembly for DirectoryServices it is not finding them.  

So basically what is needed here is to add a `_netfx` placeholder configuration into [System.DirectoryServices\ref\Configurations.props](https://github.com/dotnet/corefx/blob/master/src/System.DirectoryServices/ref/Configurations.props)

That way the reference assembly will not be built in the netfx vertical and we will be using the right reference assembly coming from the netfx targeting pack. 

However this rings me some bells since that means the System.DirectoryServices has different assembly identity in core and netfx but we're assuming they not, @ericstj is this an issue we should be concerned?

Reply:
> that means the System.DirectoryServices has different assembly identity in core 

I don't think so, at least not the last time I looked: https://github.com/dotnet/corefx/pull/21388